tests/console/commands/test_init.py

```diff
@@ -556,3 +556,13 @@ def test_predefined_and_interactive_dev_dependencies(tester, repo):
     assert expected in output
     assert 'pytest-requests = "^0.2.0"' in output
     assert 'pytest = "^3.6.0"' in output
+
+
+def test_add_package_with_extras_and_whitespace(app):
+    command = app.find("init")
+    result = command._parse_requirements(["databases[postgresql, sqlite]"])
```

Comment:
```suggestion
def test_add_package_with_extras_and_whitespace(tester):
    result = tester._parse_requirements(["databases[postgresql, sqlite]"])
```

Reply:
It look like it must be 'test._command._parse_requirements` ...